worker.go

```diff
@@ -426,4 +441,8 @@ func (tw *SpanWorker) Flush() {
 		cumulative := time.Duration(atomic.SwapInt64(&tw.cumulativeTimes[i], 0)) * time.Nanosecond
 		samples.Add(ssf.Timing(sinks.MetricKeySpanIngestDuration, cumulative, time.Nanosecond, tags))
 	}
+
+	metrics.Report(tw.traceClient, samples)
+	metrics.ReportOne(tw.traceClient,
```

Comment:
Eep! This is a per host counter with per-worker tagging meaning a rather large cardinality. I don't think this will work (or should be done) unless I'm reading it wrong.

Reply:
yeah this is hacky in the first place, to be counting the workers in this way. i don't like it, _at all_. i also don't like what it does to the interface.

but if we don't split it apart per worker, then we also can't see if the contention is spread across workers evenly or not. gotta figure out SOME way to represent this.

Reply:
or...well, i guess we can go baby steps, and i can slurp all the workers together into a single metric.

Reply:
We could deploy this as a feature branch (without merging) onto QA apiboxes alone, but I don't think we can afford that level of cardinality across all hosts Veneur runs on. :(

Reply:
cardinality issue is gone - with a single struct and multiple goroutines running against it, all the state is shared.